manifests/d/DOSBoxStaging/DOSBoxStaging/0.80.1/DOSBoxStaging.DOSBoxStaging.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.3.4 $debug=NVS1.CRLF.5-1-19041-4123.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DOSBoxStaging.DOSBoxStaging
+PackageVersion: 0.80.1
```

Comment:
@Exorcism0666 Any chance you could also submit the 0.81 update manifest as mentioned in my previous comments? :-)